src/presenced.cpp

```diff
@@ -331,6 +344,26 @@ bool Client::sendKeepalive(time_t now)
     return sendCommand(Command(OP_KEEPALIVE));
 }
 
+void Client::updatePeers(const vector<Id> &peers, bool addOrRemove)
+{
+    assert(peers.size());
+    const char *buf = mApi->sdk.getSequenceNumber();
+    Id scsn(buf);
```

Comment:
Please, add the length of `buf`.
```suggestion
    Id scsn(buf, strlen(buf));
```
or
```suggestion
    Id scsn(buf, Id::SESSIONHANDLE);
```

Reply:
Done it